### PR TITLE
Update iso-8859-15.s for º, Ö, ö

### PR DESCRIPTION
**Pull request by mobluse**
I think no characters should have identical bitmaps in 8x8, because it could lead to errors when coding or writing. There are still characters to fix, but this is a start.

### CODE REVIEW

File: charset/iso-8859-15.s [hunk 1]
```diff
@@ -2292,13 +2292,13 @@
 .byte %_██__██_
 .byte %__████__
 .byte %________
-; ö (based on PXLfont and CP850 94)
+; ö (based on CP437 94, but centered)
```
Comment (mist64) [inline charset/iso-8859-15.s]: ü will be inconsistent with ä, ö, ë etc. Why this change?

Screen Shot 2019-09-19 at 13 08 15

Comment (mobluse) [reply charset/iso-8859-15.s]: When I switched Ö to CP437 I thought I should switch ö to CP437 also, but this is perhaps a bad idea, since ä can probably not be lowered. I think the way Ö got higher would not work for the other O type characters. In that case the only way to make them different would be to lower the lower case o type character. I think we should change ö back to what it was, but it might be necessary to lower the other o type characters (except o and ö). I depends on how the upper case O type character looks. I still think it's a good idea that all characters look different.

Comment (mobluse) [reply charset/iso-8859-15.s]: When you say "based on CP850", do you then mean the 8x8 font on https://github.com/emaste/fontstuff/blob/master/syscons-fonts/cp850-8x8.hex

Comment (mist64) [reply charset/iso-8859-15.s]: I agree that all character should look different. Yes, that cp850 file. I had forgotten that the cp437 and the cp850 fonts are actually different styles. Maybe cp437 would have been a better choice for a base for style consistency with the PXLfont characters. I picked cp850 because it had more of the 8859-15 characters I needed.
